bin/jupyter-kernel.p6

```diff
@@ -48,5 +60,11 @@ multi MAIN(Bool :$generate-config!, Str :$location = default-location(), Bool :$
         note "Copying $file to $location";
         copy $resource.IO, $location.IO.child($file) or die "Failed to copy $file to $location.";
     }
-}
 
+    # Say Success
+    say "Congratulation, configuration files have been "
```

Comment:
typo -- "Congratulation" -> "Congratulations".

I like this message though, thanks :-)

Reply:
Done. Glad you like it. I think "appearances (colors) are important"